packet/Device.py

```diff
@@ -1,34 +1,80 @@
 # -*- coding: utf-8 -*-
 # SPDX-License-Identifier: LGPL-3.0-only
-
-from .OperatingSystem import OperatingSystem
+# from .Volume import Volume
 
 
 class Device:
     def __init__(self, data, manager):
         self.manager = manager
 
+        self.id = data["id"]
+        self.short_id = data["short_id"]
+        self.hostname = data["hostname"]
+        self.description = data["description"]
+        self.state = data["state"]
+        self.tags = data["tags"]
+        if "image_url" in data:
+            self.image_url = data["image_url"]
         self.billing_cycle = data["billing_cycle"]
+        self.user = data["user"]
+        self.iqn = data["iqn"]
+        self.locked = data["locked"]
+        self.bonding_mode = data["bonding_mode"]
         self.created_at = data["created_at"]
+        self.updated_at = data["updated_at"]
+        if "ipxe_script_url" in data:
+            self.ipxe_script_url = data["ipxe_script_url"]
```

Comment:
This could be tightened up to be `self.ipxe_script_url = data.get("ipxe_script_url", None)`  (this goes for the other if/else being done like this.)

Reply:
Refactored. Nice catch @mattcburns